Source/WebCore/platform/network/NetworkStorageSession.h

```diff
@@ -281,6 +291,7 @@ class NetworkStorageSession : public CanMakeWeakPtr<NetworkStorageSession> {
     RetainPtr<NSMutableSet> m_subscribedDomainsForCookieChanges;
     MemoryCompactRobinHoodHashMap<String, HashSet<CheckedPtr<CookieChangeObserver>>> m_cookieChangeObservers;
 #endif
+    HashSet<CheckedPtr<CookiesEnabledStateObserver>> m_cookiesEnabledStateObservers;
```

Comment:
We only use CheckedPtr/CheckedRef for stack variables. Please use WeakRef<> here.